pages/post.tsx

```diff
@@ -104,7 +105,7 @@ const post = () => {
           />
         )
       case 3:
-        return '스텝 4에서는 ...하세요'
+        return <EventPresent></EventPresent>
```

Comment:
별도 children을 사용하지 않는 컴포넌트로 보입니다.
닫는 태그로 명확하게 표현해주는게 좋을 것 같네요!
```suggestion
        return <EventPresent />